nessie/jobs/import_registrations.py

```diff
@@ -61,7 +61,7 @@ def run(self, load_mode='new'):
         rows = {
             'term_gpas': [],
             'last_registrations': [],
-            self.demographics_key: [],
+            'api_demographics': [],
```

Comment:
This import job always deals with API demographics even if they're not being used in the upstream merge.